### ISSUE TITLE
(Bug Report) Reels get stuck and the  accelerate itself

### ISSUE THREAD

**Issue by CySxL** [SoCuul/SCInsta#112]
When scrolling through reels, the previous/watched reels get stuck/freezed after like 2-3 seconds after playing it, and then it speeds up itself like it wanna catch up to the progress.

Makes it hard to check scrolled/watched reels again.

I've tried debugging myself by turning off features one by one and couldn't actually find the culprit. But as soon as I disable SCinsta's injection with Choicy, the reels-getting-stuck issue is gone.

Environment:
Dopamine 2 on iOS 16.3
SCInsta: 0.6.0 Rootless
Instagram: 350.0.12

**Comment by SoCuul** [comment]
Hmm, that's incredibly strange. I'm running the same Instagram version (though injected into the IPA) and have never had any similar problem occur. Do you use any other tweaks alongside SCInsta for Instagram?

**Comment by CySxL** [comment]
Yeah I'm sure I was only using SCInsta without any IG tweaks interfering.

**Comment by SoCuul** [comment]
I'll keep this issue open for now, until I've updated the tweak.

**Comment by SoCuul** [comment]
This shouldn't be an issue anymore, I'm guessing it had to do with a bug in the insta app.
